hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/components/decommissioningSummary/decommissioningSummary.tsx

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect } from 'react';
+import { AxiosError } from 'axios';
+import { Descriptions, Popover, Result } from 'antd';
+import { Datanode, SummaryData } from '@/v2/types/datanode.types';
+import { AxiosGetHelper, cancelRequests } from '@/utils/axiosRequestHelper';
+import { showDataFetchError } from '@/utils/common';
+
+type DecommisioningSummaryProps = {
+  uuid: string;
+}
+
+type DecommisioningSummaryState = {
+  loading: boolean;
+  summaryData: SummaryData | Record<string, unknown>;
+};
+
+const DecommissionSummary: React.FC<DecommisioningSummaryProps> = ({
+  uuid = ''
+}) => {
+  const [state, setState] = React.useState<DecommisioningSummaryState>({
+    summaryData: {},
+    loading: false
+  });
+  const cancelSignal = React.useRef<AbortController>();
+
+  async function fetchDecommissionSummary(selectedUuid: string) {
```

Comment:
Not sure if it is feasible, but check if we can add a loading spinner or skeleton to the UI to indicate that data is being fetched: